homeassistant/components/switchbot_cloud/cover.py

```diff
@@ -0,0 +1,233 @@
+"""Support for the Switchbot Bot as a Cover."""
```

Comment:
Last time I checked the switchbot bot was not a cover :)

Reply:
wrong comments, mistake